### ISSUE TITLE
Polarized TVS, Unpolarized TVS, Tunnel, and Varicap Diodes

### ISSUE THREAD

**Issue by Hyrumveneno** [circuitdiagram/components#42]
We have Standard, Photo, Schottky, and Light-Emitting diodes but no Polarized TVS, Unpolarized TVS, Tunnel, and Varicap Diodes.

**Comment by ThiagoRFP** [comment]
Yes, varicap and tunnel diodes are pretty standard. Would be nice to thave them. Any progress with these issues so far?

**Comment by jmed821** [comment]
Any plans on adding these?
